reference/features/configuration.md

```diff
@@ -199,6 +199,7 @@ MeiliSearch currently supports four log levels, listed in order of increasing ve
 - `'WARN:'` log all unexpected events, regardless of their severity
 - `'INFO:'` log all events. This is the default value of `--log-level`
 - `'DEBUG'`: log all events and including detailed information on MeiliSearch's internal processes. Useful when diagnosing issues and debugging
+- `'TRACE'`: log all events, and even more detailed information on MeiliSearch's internal processes. We do not advise using this level as it is overkill and overwhelming. Use `DEBUG` before considering `TRACE`.
```

Comment:
`overkill and overwhelming` has a very different tone when compared to the text in the rest of the page and is not explicit on what makes this level overwhelming. What do you think we go for this instead?

```suggestion
- `'TRACE'`: log all events and include even more detailed information on MeiliSearch's internal processes. We do not advise using this level as it is extremely verbose. Use `DEBUG` before considering `TRACE`.
```